src/index.html

```diff
@@ -5,8 +5,363 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div>
+        <a href="#">
+          <img
+            src="\images\logo.png"
+            alt="logo"
+            class="header__logo"
+          >
+        </a>
+      </div>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">apple</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link" data-qa="nav-hover">ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">гаджеты</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">планшеты</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">фото</a>
+          </li>
+
+          <li class="header__item">
+            <a href="#" class="header__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div class="card" data-qa="card">
+          <img
+            src="\images\imac.jpeg" 
+            alt="icon" 
+            class="card__img"
+            >
+          <p class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+          <p class="card__code">
+            Код товара: 195434
+          </p>
+          <div class="stars">
+            <div class="stars stars__count--4">
+              <div class="stars__star">
+              </div>
+              <div class="stars__star">
+              </div>
+              <div class="stars__star">
+              </div>
+              <div class="stars__star">
+              </div>
+              <div class="stars__star">
+              </div>
+            </div>
+        
+            <p class="stars__count">Отзывов: 5</p>
+          </div>
```

Comment:
in line 74 we have a card block element. And in line 88 - also.

in line 75 we use modifier to stars block. More detail in this [video](https://mate.academy/learn/css-advanced/bem#/video/920)

And add changes to styles according to changes in html.

Add the same changes in all cards.
```suggestion
          <div class="card__reviews">
            <div class="stars stars--4">
              <div class="stars__star">
              </div>
              <div class="stars__star">
              </div>
              <div class="stars__star">
              </div>
              <div class="stars__star">
              </div>
              <div class="stars__star">
              </div>
            </div>
        
            <p class="card__reviews-counter">Отзывов: 5</p>
          </div>
```